.gitlab-ci.yml

```
@@ -200,11 +204,29 @@ noether-cuda:
 #    Note: PETSC_DIR is set by default in GitLab runner env, unsetting to isolate core tests
     - export PETSC_DIR= PETSC_ARCH=
     - make -k -j$((NPROC_GPU / NPROC_POOL)) BACKENDS="$BACKENDS_GPU" JUNIT_BATCH="cuda" junit realsearch=%
+# Rebuild without ASAN
+    - export ASAN=0 AFLAGS= ASAN_OPTIONS=
```

```suggestion
    - unset ASAN AFLAGS ASAN_OPTIONS
```

Not significant, but a bit cleaner